recipe/meta.yaml

```diff
@@ -31,7 +29,6 @@ requirements:
     - r-base
     - tiledb 2.11.*
     - r-rcpp >=0.12.15
-    - r-testthat
```

Comment:
Excellent catch.  We once used `testthat` but simplified / lightened to may as well take advantage of it here too.